storage/s3_sdk/list_gcs_buckets.go

```diff
@@ -0,0 +1,57 @@
+
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Sample lists GCS buckets using the S3 SDK using interoperability mode.
+package main
```

Comment:
```suggestion
package s3sdk
```

`package main` needs a `func main`, and is compiled into a binary.